forest/src/daemon.rs

```diff
@@ -280,6 +293,7 @@ pub(super) async fn start(config: Config) {
         let rpc_listen = TcpListener::bind(&config.client.rpc_address)
             .await
             .unwrap_or_else(|_| cli_error_and_die("could not bind to {rpc_address}", 1));
+        set_event();
```

Comment:
If the RPC is not enable, this will block the parent process forever. If you move this below the `if` statement, it'll always work.

Reply:
Ooops. Fixing this.